WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -275,14 +233,62 @@ final class DashboardViewModel: ObservableObject {
             .filter { $0.enabled }
             .map(\.type)
         analytics.track(event: .DynamicDashboard.editorSaveTapped(types: activeCardTypes))
-        stores.dispatch(AppSettingsAction.setDashboardCards(siteID: siteID, cards: cards))
+        saveDashboardCards(cards: cards)
         dashboardCards = cards
     }
+
+    func saveDashboardCards(cards: [DashboardCard]) {
```

Comment:
nit: I think this can be a `private` method.

Reply:
Also done in 2616119ed5539e10895b5c4a12bc780b3322e2c2.